src/main/java/org/fundacionjala/trello/pages/core/WebObject.java

```diff
@@ -11,7 +12,7 @@
  */
 public abstract class WebObject {
 
-    private static final int SECONDS = 20;
+    private static final int SECONDS = Environment.getInstance().getExplicitTimeWait();
```

Comment:
I think that the Class DriverFactory should be in charge to get al the properties of the Environment, and give to webObjec through an Instance

Reply:
I'm sorry, I didn't understand your objective to do that. And I have some questions:
Why DriverFactory should have the explicit time property value?
Why does it need to wrap environment class work?

Reply:
I'm sorry, I didn't understand your objective to do that. And I have some questions:
Why DriverFactory should have the explicit time property value?
Why does it need to wrap environment class work?